crates/network-scanner-net/Cargo.toml

```diff
@@ -10,7 +10,7 @@ anyhow = "1.0.79"
 crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
 futures = "0.3.30"
 parking_lot = "0.12.1"
-polling = { git = "https://github.com/irvingoujAtDevolution/polling.git", branch = "fix-linux-tcp-fail" }
+polling = { git = "https://github.com/irvingoujAtDevolution/polling.git", branch = "fix-detect-connection-fail" }
```

Comment:
Before we can merge, we need to move this into a repository living in the Devolutions organization. You can either give the ownership of the fork to Devolutions or ask devops to clone the repo for us and push your modification there.